astropy/modeling/functional_models.py

```diff
@@ -1123,6 +1167,18 @@ def __init__(self, amplitude=amplitude.default, x_0=x_0.default,
             amplitude=amplitude, x_0=x_0, y_0=y_0, r_in=r_in, width=width,
             **kwargs)
 
+    def bounding_box_default(self):
```

Comment:
Right, if you make the image larger then the sum of the flux shouldn't change if the model is fully captured by the bounding_box. Thinking about this some more, this test doesn't necessarily require `render_model` and so it won't need custom test parameters.